.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,64 @@
+name: Bug Report
+description: Create a report to help us improve
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        Please make sure to fill out the entire form below,
+        providing as much context as you can in order to help us
+        triage and track down your bug as quickly as possible.
+
+        Before filing a bug, please be sure you have searched through
+        [existing bugs](https://github.com/open-telemetry/opentelemetry-python/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Abug)
+        to see if your bug is already addressed.
+
+        If your bug is related to an instrumentation or plugin in [opentelemetry-python-contrib](https://github.com/open-telemetry/opentelemetry-python-contrib)
+        please be sure to file it there.
+  - type: textarea
+    id: environment
+    attributes:
+      label: Describe your environment
+      description: |
+        Please describe any aspect of your environment relevant to the problem, including your Python version, [platform](https://docs.python.org/3/library/platform.html), version numbers of installed dependencies, information about your cloud hosting provider, etc. If you're reporting a problem with a specific version of a library in this repo, please check whether the problem has been fixed on main.
+      value: |
+        OS: (e.g., Ubuntu 20.04)
```

Comment:
We can bump the Ubuntu version while at it

Reply:
I think we can omit this